optimera-htb.js

```diff
@@ -265,139 +290,142 @@ function OptimeraHtb(configs) {
 
         /* ---------- Process adResponse and extract the bids into the bids array ------------*/
 
-        var bids = adResponse;
-
         /* --------------------------------------------------------------------------------- */
+        if(typeof(adResponse) !== 'undefined') {
 
-        for (var j = 0; j < returnParcels.length; j++) {
+          // put our response into an aray.
+          var bids = [adResponse][0];
```

Comment:
This essentially equals to `var bids = adResponse`, is there a reason for doing this?

Reply:
Nope, this was just older dev code while debugging. Will reset it.